README.md

```diff
@@ -1 +1,72 @@
-# Embedding-images-and-videos
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> This project was developed as a learning project part of the Microvese curriculum
+
+![screenshot](./images/screenshot.png)
+
+A simple HTML and CSS mock up of the Youtube video webpage
+
+## Live demo
+
+🔗 [YouTube Clone](https://raw.githack.com/enkog/Embedding-images-and-videos/develop/index.html)
+
+## Built With
+
+- HTML5,
+- CSS3,
+- Google Fonts
+- Font awesome
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Webbrowser and access to internet
+
+### Install
+
+There is no need to installation, just run the index.html file
+
```

Comment:
- [ ] **Mandatory:** The githack raw you included made your project look nice ✔, however, what my colleague meant with descriptive readme file can be found in this **installation section**, where you have to put the instructions to download your repo, such as, the steps in the terminal to download your project ( `git clone <the link to your repo>` ) and the steps to open a live version in localhost (clicking in the downloaded index.html file).